.rubocop/rspec.yml

```diff
@@ -25,3 +25,5 @@ RSpec/SpecFilePathFormat:
     FetchOEmbedService: fetch_oembed_service
     OEmbedController: oembed_controller
     OStatus: ostatus
+  Exclude:
+    - '**/spec/patterns/**/*'
```

Comment:
I'd prefer to not have this, but the location for the regex constants is odd. Could maybe just do the classes.